app/models/order.rb

```diff
@@ -65,6 +68,8 @@ class Order < ApplicationRecord
 
   NON_PROCESSED_STATES = ["processing", "submitted", "draft"].freeze
 
+  INTERNATIONAL_ORDERS = ["Shipment", "CarryOut"].freeze
```

Comment:
We can remove the "international orders" term completely. We don't really want that to be a thing. All orders should be processed via the same flow. You can unify the 2 validations into 1, maybe with something like this:

```ruby
  validate :verify_code_format

  def verify_code_format
    prefix = order_code_prefix(self.detail_type)
    errors.add(:base, 'Invalid code format') unless /\A#{prefix}\d{4,5}([A-Z]{1})?\z/.match(self.code)
  end
```